### PR TITLE
Avatar 

### PR DESCRIPTION
**Pull request by shubhamkumar648**
avatar

### CODE REVIEW

**Review by rahulcs754 — COMMENTED** [review]
look good

**Review by ashishnnnnn — APPROVED** [review]
Code looks good to me :+1:

File: styles/avatar.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+.img-round {
+    border-radius: 50%;
+}
+
+.avatar-codesnippet-section {
+
+    width: 100%;
+    border: 1px solid var(--primary-color);
+    background-color: #fafafa;
```
Comment (saketkum11) [inline styles/avatar.css]: use variable colors

File: styles/grow-know.style.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+}
+
+
+
+.header__brand-name a {
+
+    color: var(--primary-color);
+}
+
+
```
Comment (saketkum11) [inline styles/grow-know.style.css:52]: can  you edit the spaces.

File: styles/padding.css [hunk 3]
```diff
@@ -0,0 +1,99 @@
+/* varient padding */
+
+.p-1{
+    padding: 0.5rem; /*  8px */
+  }
```
Comment (saketkum11) [inline styles/padding.css:5]: can you add variables margin.

**Review by saketkum11 — COMMENTED** [review]
You can use the variablein margin and  padding .

**Review by aniketsingh13 — APPROVED** [review]

**Review by saketkum11 — APPROVED** [review]

File: styles/avatar.css (hunk 1 above)
Comment (shubhamkumar648) [reply styles/avatar.css]: done

File: styles/padding.css (hunk 3 above)
Comment (shubhamkumar648) [reply styles/padding.css:5]: ok

File: styles/grow-know.style.css (hunk 2 above)
Comment (shubhamkumar648) [reply styles/grow-know.style.css:52]: it make by prettier automatically

**Pull request merged**